operator/README.md

```diff
@@ -13,40 +13,29 @@ meeting to share your ideas.
 This document is an overview of how the operator works from a user perspective. For more details about the design and
 architecture and a code overview, see [ARCHITECTURE.md](./ARCHITECTURE.md).
 
-The operator CLI is distributed to users as part of istioctl. The `mesh` command
-in this repo is simply a wrapper to speed up development - the subcommands are the same code that is incorporated into
-istioctl. Making changes to any `mesh` subcommand will be reflected in istioctl after one of the regular syncs to
-istio/operator.
-
 ## Introduction
 
-This repo reorganizes the current [Helm installation parameters](https://istio.io/docs/reference/config/installation-options/) into two groups:
+The operator uses the [IstioOperator API](https://github.com/istio/api/operator/v1alpha1/operator.proto), which has
```

Comment:
maybe change it into https://github.com/istio/api/blob/master/operator/v1alpha1/operator.proto?

Reply:
also apply to all links below.

Reply:
Done.